AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,10 +491,17 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
-    stack.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(self.preferredFrameSize);
+#if DEBUG
```

Comment:
The first implementation did exactly this, it  removed all occurances of `preferredFrameSize` usage but @garrett commented on it if it would not be better to support it until we remove support for `preferredFrameSize` completely.
I think for the time being until we remove `preferredFrameSize` completly we should let it in. Let me know if you have a strong feeling against it.

Also curious of thoughts from @Adlai-Holler and @appleguy